testing/dart/canvas_test.dart

```diff
@@ -731,6 +731,27 @@ void main() {
     expect(canvas.getDestinationClipBounds(), initialDestinationBounds);
   });
 
+  test('Canvas.clipRect with matrix affects canvas.getClipBounds', () async {
+    final PictureRecorder recorder = PictureRecorder();
+    final Canvas canvas = Canvas(recorder);
+    const Rect clipBounds1 = Rect.fromLTRB(0.0, 0.0, 10.0, 10.0);
+    const Rect clipBounds2 = Rect.fromLTRB(10.0, 10.0, 20.0, 20.0);
+
+    canvas.save();
+    canvas.clipRect(clipBounds1);
+    canvas.translate(0, 10.0);
+    canvas.clipRect(clipBounds1);
+    expect(canvas.getDestinationClipBounds(), Rect.fromLTRB(0.0, 0.0, 0.0, 0.0));
```

Comment:
Same comment about measuring expectations against the "isEmpty" property rather than requiring a specific answer as I made above for the native tests.

Reply:
Done.